src/Redistribution.sol

```diff
@@ -801,11 +801,13 @@ contract Redistribution is AccessControl, Pausable {
      * @param _depth The storage depth the applicant intends to report.
      */
     function isParticipatingInUpcomingRound(address _owner, uint8 _depth) public view returns (bool) {
+        uint256 _lastUpdate = Stakes.lastUpdatedBlockNumberOfAddress(_owner);
+
         if (currentPhaseReveal()) {
             revert WrongPhase();
         }
 
-        if (Stakes.lastUpdatedBlockNumberOfAddress(_owner) >= block.number - 2 * ROUND_LENGTH) {
+        if (_lastUpdate >= block.number - 2 * ROUND_LENGTH && _lastUpdate == 0) {
```

Comment:
This is wrong again, the two conditions are never met at the same time.

```suggestion
        if (_lastUpdate >= block.number - 2 * ROUND_LENGTH || _lastUpdate == 0) {
```

Reply:
right, changing this and going back to bed. thnx for suppot